docs/containers.conf.5.md

```diff
@@ -229,6 +229,13 @@ limit is never exceeded.
 
 Default format tag for container log messages. This is useful for creating a specific tag for container log messages. Container log messages default to using the truncated container ID as a tag.
 
+**mounts**=[]
+
+List of mounts.
+Specified as "type=TYPE,source=<directory-on-host>,destination=<directory-in-container>,<options>"
```

Comment:
Can we point to the `podman-create` man page for details on supports types and options, etc?

Reply:
If we are going to start doing this, we should do it in many places, not just for Mounts, as a separate PR.